File: pkg/codegen/dotnet/testdata/third-party-package-pp/third-party-package.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using Other = ThirdParty.Other;

return await Deployment.RunAsync(() => 
{
    var Other = new Other.Thing("Other", new()
    {
        Idea = "Support Third Party",
    });

    var Question = new Other.Module.Object("Question", new()
    {
        Answer = 42,
    });

    var Question2 = new Other.Module.Sub.Object("Question2", new()
    {
        Answer = 24,
    });

    var Provider = new Other.Provider("Provider", new()
    {
        ObjectProp = 
        {
            { "prop1", "foo" },
            { "prop2", "bar" },
            { "prop3", "fizz" },
        },
    });

});

File: tests/testdata/codegen/third-party-package-pp/python/third-party-package.py
import pulumi
import third_party as other

other_1 = other.Thing("Other", idea="Support Third Party")
question = other.module.Object("Question", answer=float(42))
question2 = other.module.sub.Object("Question2", answer=float(24))
provider = other.Provider("Provider", object_prop={
    "prop1": "foo",
    "prop2": "bar",
    "prop3": "fizz",
})
